### PR TITLE
feat: Accordion and general styling. 

### PR DESCRIPTION
**Pull request by timrombergjakobsson**
Fixes: [DTS-1275](https://tegel.atlassian.net/jira/software/projects/DTS/boards/1?assignee=63ff6d6c314f5088138187f0&selectedIssue=DTS-1275):

### CODE REVIEW

**Comment by theJohnnyMe** [comment]
@timrombergjakobsson  - There seems to be an issue with the Netlify build. Please check "Latest deploy log".

File: src/app/app.component.css [hunk 1]
```diff
@@ -1,8 +1,21 @@
+@import url('@scania/tegel/dist/tegel/tegel.css');
```
Comment (theJohnnyMe) [inline src/app/app.component.css:1]: This is already imported in "styles.css".

File: src/app/app.component.css [hunk 2]
```diff
@@ -1,8 +1,21 @@
...
+@import url('https://cdn.digitaldesign.scania.com/icons/webfont/css/sdds-icons.css');
```
Comment (theJohnnyMe) [inline src/app/app.component.css:2]: Is this needed? 
Our icons come as a web-component now, so no need to import anything related to SDDS code.
Please double-check.

**Review by theJohnnyMe — CHANGES_REQUESTED** [review]

**Review by theJohnnyMe — CHANGES_REQUESTED** [review]

File: src/app/accordion-item/accordion-item.component.html [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<!-- accordion-item.component.html -->
+<div [class.expanded]="expanded">
```
[the remark this answers is not in the document]
Comment (timrombergjakobsson) [reply src/app/accordion-item/accordion-item.component.html]: No, your right, was a piece of old code that was to be removed. Missed it.

**Review by theJohnnyMe — APPROVED** [review]
Nice work buddy!

**Pull request merged**